README.md

```diff
@@ -24,7 +24,7 @@ using a build system, please make sure these dependencies are on the classpath.
 
 # Prerequisites
 
-**If your project depends on the Snowflake JDBC driver, as well, please make sure the JDBC driver version is 3.13.30 to 3.13.33. JDBC driver version 3.14.0 or higher is currently not supported. **
+**If your project depends on the Snowflake JDBC driver, as well, please make sure the JDBC driver version is 3.13.30 to 3.14.5.
```

Comment:
```suggestion
**If your project depends on the Snowflake JDBC driver, as well, please make sure the JDBC driver version is 3.13.30 or above.
```

Reply:
I would be explicit about the versions we support. Sometimes a new JDBC version comes out and we have not tried it.

This happened with a previous JDBC version and I made a change here to call out the explicit versions

Reply:
+1

Reply:
yeah, but this is less ideal, what's the point of NOT shading the JDBC jar if we keep running into issues with other JDBC versions? If it's shaded, then we can always use a version without any issue

Reply:
but still we use some JDBC with an ingest SDK right? which we feel is safe to use?

Reply:
I suggest we postponed this discussion and keep this as is

Reply:
keep what you have in the PR :)